conanfile.py

```diff
@@ -41,8 +41,10 @@ def build(self):
 
     def package(self):
         self.copy(pattern="LICENSE.txt", dst="licenses", src=self._source_subfolder)
+        self.copy(pattern="examples/imgui_impl_*", dst="misc/bindings", src=self._source_subfolder, keep_path=False)
         cmake = self._configure_cmake()
         cmake.install()
 
     def package_info(self):
+        self.cpp_info.srcdirs = ["misc/bindings", ]
```

Comment:
Would be better `self.cpp_info.srcdirs = ["misc", ]`?

Reply:
Yes, I think so.

Reply:
According to the package, we only export the `misc/fonts`, so `misc/bindings` does not make sense